themes/survey/fruity_twentythree/views/layout_survey_list.twig

```diff
@@ -176,20 +176,20 @@ NOTE: Feel free to ask for functions, variables, etc, that you feel are missing
                                 </div>
                             </div>
                         </div>
-                    </div>
+                    </main>
 
-                    <div id="{{ aSurveyInfo.id.surveylistfooter }}"
+                    <footer role="contentinfo" id="{{ aSurveyInfo.id.surveylistfooter }}"
```

Comment:
Must be documented (in my opinion) on https://github.com/LimeSurvey/LimeSurvey/blob/master/themes/survey/fruity_twentythree/views/subviews/footer/footer.twig

Have 2 footer are a bad practice

Reply:
Yeah, it is not ideal, but  in general several footers are allowed.

Reply:
If you have <section> here the section are the body.

Her : it's just adding a comment on existing footer.twig file, no issue bu can be interseting